default/python/AI/PolicyAI.py

```diff
@@ -116,7 +116,7 @@ def __init__(self, status_only: bool = False):
         self._populated_planet_ids = PlanetUtilsAI.get_populated_planet_ids(empire_owned_planet_ids)
         self._num_populated = len(self._populated_planet_ids)
         self._num_outposts = len(empire_owned_planet_ids) - self._num_populated
-        self._max_turn_bureaucracy = self._calculate_max_turn_bureaucracy()
+        self._max_turn_bureaucracy = 9999
```

Comment:
`_calculate_max_turn_bureaucracy` is not used anymore.  Maybe move 9999 inside it? 

@Grummel7 , whom would be the best person to review FOCS changes?

Reply:
Well, this is just a quick fix to avoid the AI from regularly replacing bureaucracy.
Once the PR is merged, I want to overhaul the entire centralization / bureaucracy stuff.

Usually Geoff does checks this kind of FOCS changes, but I guess at least Oberlus or agrrr3 could review the FOCS change as well.

Reply:
> Well, this is just a quick fix to avoid the AI from regularly replacing bureaucracy.

leaving dead code is bad practice, even if you have a plan to deal with it later.    So let's just move 9999 inside and add a comment on why it is so.     

Basically, things that you said in that comment could be just a part of the code.